UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -958,14 +958,19 @@ public void CmdFilledDynamicInput(GameObject forGameObject, string input)
 	}
 
 	[Command]
-	public void CmdTriggerStorageTrap(IEnumerable<ItemSlot> slots, GameObject player)
+	public void CmdTriggerStorageTrap(GameObject storage)
 	{
-		foreach (var slot in slots)
+		//Probably want to put a validations check here to make sure backpack is in range
+		//though this is only gonna hurt this player so isnt really hackable lol
+		if(storage == null) return;
+		if(storage.TryGetComponent<DynamicItemStorage>(out var slots) == false) return;
```

Comment:
I think you just want this to be `TryGetComponent<InteractableStorage>`, pretty sure DynamicItemStorage is just on the player object

Reply:
should be solved now